state-transition/core/state_processor_forks.go

```diff
@@ -136,7 +135,7 @@ func (sp *StateProcessor) logDeneb1Fork(
 	// Since state fork is not updating to Deneb1, every block observes Deneb1 as "new fork" during
 	// Deneb1. Hence, we must wrap this in a OnceFunc to ensure it is logged only the first time
 	// we process a Deneb1 block.
-	sync.OnceFunc(func() {
```

Comment:
the OnceFunc was being re-triggered every time this function is called and never saved for re-use.